src/components/default-error-fallback.tsx

```diff
@@ -88,48 +90,64 @@ const RightPanel = () => {
 };
 
 export default function DefaultErrorFallback() {
+	const { isRTL } = useI18n();
+
+	useEffect( () => {
+		document.documentElement.dir = isRTL() ? 'rtl' : 'ltr';
```

Comment:
When Studio displays default error fallback, translations are not available, so it will always switch to LTR. We could even simplify this code and hardcode it to `ltr`. Any thoughts?

Reply:
I think it's better if we hard code it to LTR. That way we don't have "dead" code that won't work.